students/saleta_sebastian/lesson_12_debugging/regex_search.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import logging
+import os
+import re
+import sys
+
+
+logging.basicConfig(level=logging.DEBUG, format=(' %(asctime)s - %(levelname)s'
+                                                 ' - %(message)s'))
+logger = logging.getLogger('__name__')
+
+
+def set_logger_level(lvl):
+    if lvl == 0:
+        logging.disable(logging.CRITICAL)
+    elif lvl == 1:
+        logger.setLevel('WARNING')
+    elif lvl == 2:
+        logger.setLevel('INFO')
+
+
+def check_arg(args=None):
+    parser = argparse.ArgumentParser(description='Script to find sentence'
+                                                 ' in txt files ')
+    parser.add_argument('-p', '--sentence',
```

Comment:
You like this kind of mistakes ```-p``` vs ```-s```

Reply:
yahh i think that was copied from last lesson 🤕